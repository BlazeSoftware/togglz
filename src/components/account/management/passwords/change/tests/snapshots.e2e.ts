import snapIt from '../../../../../../test/snap-it';

describe('delete environment', () => {
  const user = {
    email: 'another@dot.com',
  };

  snapIt('account-change-password')(
    'renders correctly',
    `<account-change-password user=${user}></account-change-password>`
  );
});
