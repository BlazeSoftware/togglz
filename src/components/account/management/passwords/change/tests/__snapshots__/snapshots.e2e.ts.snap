// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`delete environment renders correctly 1`] = `
"<account-change-password user=\\"[object\\" object]=\\"\\" class=\\"hydrated\\"><blaze-drawer class=\\"hydrated\\"><!----><div aria-hidden=\\"true\\" class=\\"c-overlay c-overlay--fullpage\\"></div><aside aria-expanded=\\"async isOpen() {
        return this._isOpen;
    }\\" class=\\"o-drawer o-drawer--right\\"><button type=\\"button\\" class=\\"c-button c-button--close\\">Ã—</button><blaze-card class=\\"hydrated\\"><!----><div class=\\"c-card\\"><form><blaze-card-header class=\\"hydrated\\"><!----><header class=\\"c-card__header\\"><h2 class=\\"c-heading\\">Change password</h2></header></blaze-card-header><blaze-card-body class=\\"hydrated\\"><!----><div class=\\"c-card__body\\"><blaze-alert class=\\"hydrated\\"><!----><div role=\\"alert\\" hidden=\\"\\" class=\\"c-alert u-display-none\\"><div></div><div></div></div></blaze-alert><label class=\\"o-form-element c-label\\">Current:<div class=\\"c-input-group c-input-group--label\\"><div class=\\"o-field o-field--icon-left\\"><i aria-hidden=\\"\\" class=\\"fa-fw fas fa-lock c-icon\\"></i><input type=\\"password\\" class=\\"c-field\\" required=\\"\\" minlength=\\"6\\"></div><button type=\\"button\\" class=\\"c-button c-button--ghost c-button--brand\\">Show</button></div></label><label class=\\"o-form-element c-label\\">New:<div class=\\"c-input-group c-input-group--label\\"><div class=\\"o-field o-field--icon-left\\"><i aria-hidden=\\"\\" class=\\"fa-fw fas fa-lock c-icon\\"></i><input type=\\"password\\" class=\\"c-field\\" required=\\"\\" minlength=\\"6\\"></div><button type=\\"button\\" class=\\"c-button c-button--ghost c-button--brand\\">Show</button></div></label></div></blaze-card-body><blaze-card-footer class=\\"hydrated\\"><!----><footer class=\\"c-card__footer\\"><button class=\\"c-button c-button--block c-button--success\\"><span class=\\"c-button__icon-left\\" aria-hidden=\\"\\"><i aria-hidden=\\"\\" class=\\"fa-fw fas fa-save\\"></i></span>Save password</button></footer></blaze-card-footer></form></div></blaze-card></aside></blaze-drawer></account-change-password>"
`;
